tests/test_regularization.py

```diff
@@ -85,30 +92,31 @@ def test_cutmixd(self):
             label = torch.randint(0, 1, shape)
             sample = {"a": t, "b": t, "lbl1": label, "lbl2": label}
             cutmix = CutMixd(["a", "b"], 6, label_keys=("lbl1", "lbl2"))
+            cutmix.set_random_state(seed=123)
             output = cutmix(sample)
-            # croppings are different on each application
-            self.assertTrue(not torch.allclose(output["a"], output["b"]))
```

Comment:
Behaviour changed here since for dictionary version, it will apply all the same parameters on different keys. It will align with other transforms.

Reply:
Yes we can deterministically inspect the results now.